skyportal/tests/tools/test_offset_util.py

```diff
@@ -1,10 +1,110 @@
+import uuid
+
 import pytest
+import numpy as np
+import numpy.testing as npt
 import requests
 from requests.exceptions import HTTPError, Timeout, ConnectionError
 
-from skyportal.utils import get_nearby_offset_stars, get_finding_chart, get_ztfref_url
-
+from skyportal.tests import api
+from skyportal.utils import (
+    get_nearby_offset_stars,
+    get_finding_chart,
+    get_ztfref_url,
+    calculate_best_position,
+)
 from skyportal.utils.offset import irsa
+from skyportal.models import Photometry
+
+
+def test_calculate_best_position_no_photometry():
+
+    ra, dec = calculate_best_position(
+        [], fallback=(10.0, -20.0), how="snr", max_offset=0.5, sigma_clip=4.0
+    )
+    npt.assert_almost_equal(ra, 10)
+    npt.assert_almost_equal(dec, -20)
+
+
+def test_calculate_best_position_with_photometry(
```

Comment:
Maybe mark this as flaky? Failed on Travis but seems fine locally.

Reply:
Done